lib/matplotlib/tests/test_agg.py

```diff
@@ -250,6 +250,9 @@ def test_pil_kwargs_tiff():
 
 
 def test_pil_kwargs_webp():
+    Image.init()
+    if "WEBP" not in Image.SAVE:
+        pytest.skip("No WEBP support")
```

Comment:
It looks like Pillow has 3.12 wheels now; maybe we don't need to skip this?

Reply:
I'll see if it skips the tests after rebasing. If not, I will remove it.